addons/knowledge/static/src/webclient/commands/knowledge_providers.js

```diff
@@ -95,6 +95,7 @@ commandProviderRegistry.add("knowledge", {
 
             },
             category: "knowledge_articles",
+            href: `/knowledge/article/${article.id}`,
```

Comment:
I think it was done on purpose, so that one can share the links with frontend/portal users when right clicking in the CP and copying the link address.
From what I see in the task's log, it seems to have been discussed between faba and lna